src/components/DialogWithCloseButton/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { Dialog, DialogProps, IconButton, DialogTitle } from '@mui/material';
-import CloseIcon from '@mui/icons-material/Close';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons/faClose';
```

Comment:
```suggestion
import faClose from '@fortawesome/free-solid-svg-icons/faClose';
```

Reply:
OR

```suggestion
import { faClose } from '@fortawesome/free-solid-svg-icons';
```

Reply:
@JSReds thanks, I'll try that. FYI I picked the import syntax from FA docs since they seem to say it's better for tree shaking: https://docs.fontawesome.com/web/use-with/react/add-icons